ddp-workspace/projects/ddp-dsm-ui/src/app/patients/patients.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { AddPatientsModalComponent } from '../add-patients-modal/add-patients-modal.component';
+
+@Component({
+  selector: 'app-patients',
+  templateUrl: './patients.component.html',
+  styleUrls: ['./patients.component.scss'],
+})
+export class PatientsComponent implements OnInit {
```

Comment:
The participants/patients component with real data already exist in develop branch (implemented by Giorgi).
Can you please integrate your code with it?